src/App.js

```diff
@@ -53,7 +53,7 @@ export const parseUrl = (
   if (!parsed) {
     return null;
   }
-  const [_, service, product, version] = parsed;
+  const [, service, product, version] = parsed;
```

Comment:
oh, wait, you can do that?!

Reply:
It's nuts isn't it! Why I first read about it I was surprised too. 
https://codepen.io/peterbe/pen/eVJRVK?editors=1111